src/pipeline/modules/index/indicies.py

```diff
@@ -5,4 +5,11 @@
 """
 class Indicies(Enum):
     NDVI = "NDVI",
-    NUTRIENT = "NUTRIENT"
\ No newline at end of file
+    NUTRIENT = "NUTRIENT",
+    DOUBLEPLANT = "DOUBLEPLANT",
+    DRYDOWN = "DRYDOWN",
+    ENDROW = "ENDROW",
+    PLANTERSKIP = "PLANTERSKIP",
+    WATER = "WATER",
+    WATERWAY = "WATERWAY",
+    WEEDCLUSTER = "WEEDCLUSTER"
```

Comment:
Can you add very short descriptions of the indices, like:

```
"Normalized Differential Vegetation Index. Suggests whether the area contains live vegetation"
NDVI = "NDVI"
```

Reply:
sure